app/Bookstore.Domain/Books/Book.cs

```diff
@@ -42,15 +43,19 @@ public class Book : Entity
 
         public string ISBN { get; set; }
 
+        [JsonIgnore]

```

Comment:
Same as above. If we can remove these attributes it would be good.

Reply:
Done